src/templates/react/src/helper/index.js

```diff
@@ -1,3 +1,3 @@
-export const isEmpty = (obj) =>
-	[Object, Array].includes((obj || {}).constructor) &&
-	!Object.entries(obj || {}).length
\ No newline at end of file
+
+import { isEmpty } from './isEmpty'
+export default isEmpty
```

Comment:
import { isEmpty } from './isEmpty'
=>
import isEmpty from './isEmpty'
isEmpty should be a default export from './isEmpty'

whereas helper function would export multiple functions so no default needed in it
export {
isEmpty
}

<img width="364" alt="image" src="https://user-images.githubusercontent.com/94058485/191897113-073ee042-2077-4222-89de-4ad11d4069c9.png">